src/api/send/test.rs

```diff
@@ -231,7 +223,7 @@ fn missing_subject_field() {
         .header(ContentType::JSON)
         .body(
             r#"{
-      "to": [ "foo@example.com" ],
+      "to": "foo@example.com",
```

Comment:
And here, because we didn't have the error detail mentioned above, we hadn't actually noticed that a bunch of these tests were set-up wrong, and were failing for the wrong reason.